__tests__/command_helpers/getSolidaritySettings.ts

```diff
@@ -0,0 +1,31 @@
+import { read } from 'fs';
+
+import { solidarity } from '../../src';
+import getSolidaritySettings from '../../src/extensions/functions/getSolidaritySettings'
+
+import solidarityExtension from '../../src/extensions/solidarity-extension'
+
+const context = require('gluegun')
+solidarityExtension(context)
+
+describe('getSolidaritySettings', () => {
+  describe('w/ .solidarity', () => {
+    test('happy path', () => {
+      const solidaritySettings = getSolidaritySettings(context)
+      expect(solidaritySettings).toMatchSnapshot()
+    })
+
+    // Need to figure out a good pattern for mocking filesystem  
+    // to point at our .solidarity.example 
+
+    // test('sad path', () => {
```

Comment:
I think I have 1 sad path test in place.  Take a look at the conflict.  It's not a great solution, so feel free to improve it.

Reply:
@GantMan It does test the case where there is no .solidarity file so that works. Another thought would be to test the case where the .solidarity file is bad json and can't be parsed.